client/lib/products-values/translations.js

```diff
@@ -86,6 +96,21 @@ export const getJetpackProductsDisplayNames = () => {
 			i5: translate( 'Site Search' ),
 		}[ currentCROvariant ] || translate( 'Jetpack Search' );
 	const scan = currentCROvariant === 'i5' ? translate( 'Scan' ) : translate( 'Jetpack Scan' );
+	const scanRealtime = (
+		<>
+			{ currentCROvariant === 'i5'
+				? translate( 'Scan {{em}}Real-Time{{/em}}', {
+						components: {
+							em: <em />,
+						},
+				  } )
+				: translate( 'Jetpack Scan {{em}}Real-Time{{/em}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Scan {{em}}Real-time{{/em}}' )` *ES Score: 9*